node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -175,6 +175,26 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       "Deploy a Rholang source file to Casper on an existing running node. " +
         "The deploy will be packaged and sent as a block to the network depending " +
         "on the configuration of the Casper instance.")
+
+    val addressCheck: String => Boolean = addr =>
+      addr.startsWith("0x") && addr.drop(2).matches("[0-9a-fA-F]+")
+    val from = opt[String](
+      descr = "Purse address that will be used to pay for the deployment.",
+      validate = addressCheck
+    )
+
+    val phloLimit = opt[Int](
+      descr = "The amount of phlo to use for the transaction (unused phlo is refunded)."
+    )
+
+    val phloPrice = opt[Int](
+      descr = "The price of gas for this transaction in phlo."
```

Comment:
Should be "the price of phlo for this transaction in units phlo/rev".

Remember phlo <-> gas and rev <-> wei (or possibly eth... I'm not sure about denominations here, @guardbotmk3 @pyrocto @nashef comments?)

Reply:
As per discussion: rev <-> eth and "dust" <-> wei
Please update the comments/descriptions appropriately @goral09